docs/platforms/javascript/guides/sveltekit/manual-setup.mdx

```diff
@@ -66,7 +66,7 @@ Sentry.init({
 
 3. Add the `handleErrorWithSentry` function to the [`handleError` hook](https://kit.svelte.dev/docs/hooks#shared-hooks-handleerror):
 
-```javascript {filename:hooks.client.js}
+```javascript {filename:hooks.client.js} {5}
```

Comment:
what does the `{5}` do? 😅

Reply:
Highlight a line in the code snippet:

![image](https://github.com/getsentry/sentry-docs/assets/8420481/d0e9fbd3-8a9f-4484-b10d-607734f27f8e)

Super happy that this is finally possible in docs